openshift/patches/023-apiserversource-configmap-trusted-cabundle.patch

```diff
@@ -126,9 +126,9 @@ index 6b19bbbb0e1..40148851e34 100644
  					Containers: []corev1.Container{
  						{
  							Name:  "receive-adapter",
-@@ -109,6 +129,13 @@ func MakeReceiveAdapter(args *ReceiveAdapterArgs) (*appsv1.Deployment, error) {
- 								RunAsNonRoot:             ptr.Bool(true),
+@@ -112,6 +132,13 @@ func MakeReceiveAdapter(args *ReceiveAdapterArgs) (*appsv1.Deployment, error) {
  								Capabilities:             &corev1.Capabilities{Drop: []corev1.Capability{"ALL"}},
+ 								SeccompProfile:           &corev1.SeccompProfile{Type: corev1.SeccompProfileTypeRuntimeDefault},
```

Comment:
arent we removing this w/ the other patch?

Historically we have had the remove-seccomp patch _before_ the `OcpTrusedCaBundleMountPath` bundle path

But... if this works, I am OK w/ that

Reply:
As the other patches have a numeric prefix those get executed first: https://github.com/openshift-knative/eventing/blob/22458235ea6c549ef84aa9f805b8606e4607a09d/openshift/release/update-to-head.sh#L29-L35

Also from Jenkins:
```
...
00:04:14.743   create mode 100644 openshift/tls/issuers/eventing-ca-issuer.yaml
00:04:14.743   create mode 100644 openshift/tls/issuers/selfsigned-issuer.yaml
00:04:14.743  Applying patch openshift/patches/013-sinkbinding_webhook_corrections.patch
00:04:14.743  Applying patch openshift/patches/015-sinkbinding_v1_tests_skip.patch
00:04:14.743  Applying patch openshift/patches/017-SourceCRDControllerFilterNonDucks.patch
...
```